config/initializers/008-rack-cors.rb

```diff
@@ -32,9 +32,8 @@ def call(env)
 
   def self.apply_headers(cors_origins, env, headers)
     request_method = env['REQUEST_METHOD']
-    cdn_endpoints = ["/assets", "/javascripts"]
 
-    if cdn_endpoints.include?(env['SCRIPT_NAME']) && Discourse.is_cdn_request?(env, request_method)
+    if env['REQUEST_PATH'] =~ /\/(javascripts|assets)\// && Discourse.is_cdn_request?(env, request_method)
```

Comment:
I kind of have some reservation about this change partly because I don't know what kind of request paths we are dealing with. Maybe @SamSaffron can review this?

Reply:
It's adding CORS header while requesting the public files like [/public/javascripts/jquery.magnific-popup.min.js](https://github.com/discourse/discourse/blob/48c0f3faf35d54c48bdeef0c44739dc94d1abff5/public/javascripts/jquery.magnific-popup.min.js).